extensions/parquet/table/src/main/java/io/deephaven/parquet/table/pagestore/OffsetIndexBasedColumnChunkPageStore.java

```diff
@@ -139,6 +163,24 @@ private ChunkPage<ATTR> getPageImpl(@Nullable FillContext fillContext, int pageN
     @Override
     @NotNull
     public ChunkPage<ATTR> getPageContaining(@Nullable final FillContext fillContext, long rowKey) {
+        if (isInitialized) {
+            return getPageContainingImpl(fillContext, rowKey);
+        }
+        // We need to initialize this class
+        if (fillContext != null) {
+            initialize(fillContext);
+            return getPageContainingImpl(fillContext, rowKey);
+        }
+        // Create a fill context and use it to initialize this class as well as read the page
+        // TODO What should be the chunk capacity here?
+        try (final FillContext fillContext1 = makeFillContext(1, null)) {
+            initialize(fillContext1);
+            return getPageContainingImpl(fillContext1, rowKey);
+        }
+    }
```

Comment:
Please suggest if there is a better way to do this, also what should I keep the chunk capacity here.